core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -1002,4 +1014,38 @@ public void createCronForUserHomeCreation(StackDto stackDto, Set<String> candida
             throw new CloudbreakException(message, e);
         }
     }
+
+    private void addRangerRazServiceIfAvailable(StackDto stackDto, String cmVersion, Map<String, List<String>> serviceLocations, GatewayConfig gatewayConfig) {
+        if (stackDto.getCluster() != null && stackDto.getStack() != null
```

Comment:
I think logic can come from the RangerRazDatahubConfigProvider? Probably with some refactor we can easily use the same logic here. (no dup needed here)

Reply:
Done